README.md

```diff
@@ -49,15 +49,25 @@ You can use the create react app setup in `playground/nextjs` to test posthog-js
 
 ## Developing together with another project
 
-Install pnpm to link a local version of `posthog-js` in another JS project: `npm install -g pnpm` 
+Install pnpm to link a local version of `posthog-js` in another JS project: `npm install -g pnpm`
 
-#### Run this to link the local version
+### Run this to link the local version
 
+We have 2 options for linking this project to your local version: via [pnpm link](https://docs.npmjs.com/cli/v8/commands/npm-link) or via [local paths](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#local-paths)
+
+#### local paths (preferred)
+
+- from whichever repo needs to require `posthog-js`, go to the `package.json` of that file, and replace the `posthog-js` dependency version number with `file:<relative_or_absolute_path_to_local_module>`
+- e.g. from the `package.json` within `posthog`, replace `"posthog-js": "1.131.4"` with `"posthog-js": "file:../posthog-js"`
```

Comment:
stupid question... this is the path to the git repo?

Reply:
It is not, it's the path to your locally installed instance.  See https://docs.npmjs.com/cli/v9/configuring-npm/package-json#local-paths.